### PR TITLE
Dendro ie render fix

### PR DESCRIPTION
**Pull request by xjackk**
* https://federal-spending-transparency.atlassian.net/browse/DA-5390

### CODE REVIEW

File: src/components/visualizations/dendro/dendro.jsx [hunk 1]
```diff
@@ -59,8 +59,8 @@ function Dendro(props) {
       baseSvg.call(zoomListener).on("wheel.zoom", null); // dont let window scroll interfere with scroll for viz. 
 
       // Append a group which holds all nodes and which the zoom Listener can act upon.
-      svgGroup = baseSvg.append('g').attr('id', function() {
-        return'dendro-viz';
+      svgGroup = baseSvg.append('g').attr('id', function () {
+        return 'dendro-viz';
```
Comment (brett-varney) [inline src/components/visualizations/dendro/dendro.jsx]: Again, why is this a function that returns a constant?

**Comment by brett-varney** [comment]
comment

File: src/components/visualizations/dendro/dendro.jsx [hunk 2]
```diff
@@ -501,27 +494,8 @@ function Dendro(props) {
...
-        theSvg.dispatchEvent(event);
-        event.initMouseEvent('mousemove', true, true, window, 0, 0,
-          0, 2, 2, false, false, false, false,
-          0, null);
-        theSvg.dispatchEvent(event);
-        event.initMouseEvent('mouseup', true, true, window, 0, 0,
-          0, 2, 2, false, false, false, false,
-          0, null);
-        theSvg.dispatchEvent(event);
-      }
```
Comment (brett-varney) [inline src/components/visualizations/dendro/dendro.jsx]: Thanks for removing code that is no longer used.

File: src/components/visualizations/dendro/dendro.jsx [hunk 3]
```diff
@@ -533,78 +507,77 @@ function Dendro(props) {
...
-            d3.select('#tree-container').append('html')
-              .attr('width', viewerWidth)
-              .attr('height', viewerHeight)
-              .attr("viewBox", `0 0 ${viewerWidth} ${viewerHeight}`)
-              .attr('id', 'svg-dendrogram')
-              .attr('class', 'overlay')
-              .html('<h1>We will update the Federal Account Explorer as soon as data is available</h1>');
-          }
+      let FiscalYear = $('input[name="FiscalYear"]:checked').val();
+      let Quarter = $('input[name="Quarter"]:checked').val();
```
Comment (brett-varney) [inline src/components/visualizations/dendro/dendro.jsx]: Do these values ever change? const catches trying to change a variable that shouldn't.

File: src/containers/fed-acct/fed-acct-container.jsx [hunk 4]
```diff
@@ -11,12 +11,14 @@ class FedAcctContainer extends React.Component {
...
       fiscalYearVal: 'fy19', // default fy19
+      quarterVal: '12-31' // default 12-31 quarter (1st)
     };
   };
 
   radioCheck = (event) => {
     this.setState({
-      fiscalYearVal: event.target.value
+      fiscalYearVal: event.target.value,
+      quarterVal: event.target.value
```
Comment (matthewbrown314) [inline src/containers/fed-acct/fed-acct-container.jsx]: quarterVal isn't used anywhere in this file. Is this necessary?

**Comment by brett-varney** [comment]
@xjackk Builds & works, just a couple of questions/changes

**Comment by xjackk** [comment]
@matthewbrown314 @brett-varney solved all the things you guys pointed out - thanks

**Pull request merged**